### ISSUE TITLE
Day planner doesn't play well with Dropbox - hundreds of conflicting copies

### ISSUE THREAD

**Issue by danieltomasz** [ivan-lednev/obsidian-day-planner#70]
I am using Day Planner with Vault that  is shared in Dropbox between different computers.
Something is causing dropbox to create hundreds of conflicting copies of my Day planner when I open my vault on another computer, I didn't edited Day Planner on another computer.

The problem usually appears when two computers had open the same vault at the same time.

I didn't use the Mermaid chart for that day.
Obsidian 0.11.5
Day Planner 0.5.8

**Comment by takeo** [comment]
I ran into this too. When I was using iCloud drive to sync I was getting a lot of sync conflicts since it seems like the plugin saves a new version every few seconds. Even while my other laptop was asleep, just having Obsidian running on both machines caused problems. I've since moved to Obsidian sync, which is great, but even only having Day Planner on one machine seems to create a lot of noise in the sync logs, with lots of these messages:

`Rejected server change Daily Notes /2021-04-28.md`
